File: .repos/nexmo-community/nexmo-java-quickstart/src/main/java/com/nexmo/quickstart/initialize/BasicAuth.java
package com.nexmo.quickstart.initialize;

import com.nexmo.client.NexmoClient;

import static com.nexmo.quickstart.Util.envVar;

/**
 * Example of configuring a NexmoClient with an API secret.
 */
public class BasicAuth {
    public static void main(String[] argv) throws Exception {
        String NEXMO_API_KEY = envVar("NEXMO_API_KEY");
        String NEXMO_API_SECRET = envVar("NEXMO_API_SECRET");

        NexmoClient client = new NexmoClient.Builder().apiKey(NEXMO_API_KEY).apiSecret(NEXMO_API_SECRET).build();
    }
}

File: .repos/nexmo-community/nexmo-dotnet-quickstart/NexmoDotNetQuickStarts/Authentication/BasicAuth.cs
using Nexmo.Api;

namespace NexmoDotNetQuickStarts.Authentication
{
    /// <summary>
    /// This class is only used for the purpose of building documentation on https://developer.nexmo.com/
    /// It is not used in any of the code samples provided in these Quickstarts
    /// </summary>
    public class BasicAuth
    {
        public BasicAuth()
        {
            var client = new Client(creds: new Nexmo.Api.Request.Credentials
            {
                ApiKey = "NEXMO_API_KEY",
                ApiSecret = "NEXMO_API_SECRET"
            });
        }
    }
}